src/app/rundown-view/components/rundown/rundown.component.ts

```diff
@@ -64,6 +84,9 @@ export class RundownComponent implements OnInit, OnDestroy {
 
   public ngOnDestroy(): void {
     this.rundownTimingContextSubscription?.unsubscribe()
+    for (let rindownEventSubscription of this.rindownEventSubscriptions) {
+      rindownEventSubscription?.unsubscribe()
+    }
```

Comment:
You could write this as `this.rindownEventSubscriptions.forEach(subscription => subscription.unsubscribe)`

Reply:
Done